receiver/mongodbreceiver/metadata.yaml

```diff
@@ -29,6 +29,14 @@ attributes:
       - delete
       - getmore
       - command
+  operation_latency:
+    name_override: operation
+    description: The MongoDB operation with regards to latency
+    type: string
+    enum:
+      - reads
+      - writes
+      - commands
```

Comment:
```suggestion
      - read
      - write
      - command
```

Reply:
I used the plural form because mongodb returns:

```
{
	"reads": {
		"latency": new NumberLong("0"),
		"ops": new NumberLong("0")
	},
	"writes": {
		"latency": new NumberLong("0"),
		"ops": new NumberLong("0")
	},
	"commands": {
		"latency": new NumberLong("1890"),
		"ops": new NumberLong("24")
	},
	"transactions": {
		"latency": new NumberLong("0"),
		"ops": new NumberLong("0")
	}
}
```

Otherwise, https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16854/files#diff-eeb8431ccc1f61e78c1c3bee9c5ec90e18bf2a3a95b27d1fe7543f466d56d91bR221 would a small formatting change. I guess the preference for singular form is for semantics.

Reply:
Yeah, singular is the preferred in OTel. In most cases we will prefer our own semantic conventions over those established by other systems, since if we defer too much to other systems we will have very irregular naming. The exception would be in cases where there is a very specific terminology defined by the technology that would be unclear if changed.